tests/testMask.py

```diff
@@ -349,7 +349,7 @@ def testInterpret(self):
             self.assertEqual(im.interpret(bitmask), p)
             im.getArray()[0, i] = bitmask
             self.assertEqual(im.getAsString(i, 0), p)
-        self.assertEqual(self.Mask.interpret(allBits), ",".join(planes.keys()))
+        self.assertEqual(self.Mask.interpret(allBits), ",".join(sorted(planes.keys())))
```

Comment:
I remember a discussion about this on slack but I can't recall the final conclusion. I know that planes is supposed to be a dict with sorted keys, can you remind me (and future developers) why it is ok that planes is not sorted anymore?